msix/ubuntu-pro-agent-launcher/main.cpp

```diff
@@ -30,6 +30,9 @@ std::filesystem::path thisBinaryDir() {
 }
 
 int WINAPI wWinMain(HINSTANCE, HINSTANCE, PWSTR pCmdLine, int) try {
+  // Request to restart if closed for installing updates.
+  RegisterApplicationRestart(
+      NULL, RESTART_NO_CRASH | RESTART_NO_HANG | RESTART_NO_REBOOT);
```

Comment:
We're ignoring the possible return value. I suppose it's fine as we cannot really do anything about it. We should consider it in the future if we improve our logging solution.